### ISSUE TITLE
[feature request] Research contributors get notified about interactions with Research article

### ISSUE THREAD

**Issue by thisislawatts** [ONEARMY/community-platform#2143]
**Describe the solution you'd like**
Based on the work in #2006, it will be possible for multiple users to be associated with a Research article. The intent here is to facilitate a discussion between the research contributors and community members interacting with the article.

The expected behaviour here is that all notifications a Research Article author would receive are also sent to tagged contributors. A `notification` is the items displayed in the NotificationsList component. 

For example:

* If a user marks a Research article as Useful, each of the contributors will receive that notification.
* If a user comments on a Research article, each of the contributors will receive that notification. 

Out of scope:

* This Issue does **not** cover sending email notifications. There is [a separate thread of work around this functionality](https://github.com/ONEARMY/community-platform/issues/2091).

**Describe alternatives you've considered**
None.

**Comment by thisislawatts** [comment]
Adding Blocked label until #2007 is merged.

**Comment by davehakkens** [comment]
unblocked? :)

**Comment by thisislawatts** [comment]
Seems like it @davehakkens, in theory the code that adds this has already been merged. Will do some manual QA to see if its available on `master`.

**Comment by thisislawatts** [comment]
Have done manual QA on this in `master` with the following steps.

1. Created a [new piece of Research](http://localhost:3000/research/research-notifications-for-contributors)
2. Added User B as collaborator
3. User A added comment to Research
4. Verified User B received a new comment notification linking back to the Research.
